### DISCUSSION TITLE
Parsing ASB files?

### DISCUSSION THREAD

**Discussion by MediaMoots** [api-token/ainb#1] [General]
Is it possible to parse ASB files yet? Or is ASB very different from AINB?

**Comment by api-token** [comment]
I haven't looked much into ASB files yet. There does seem to be some similarities though. Once I'm finished with AINB files, I'll look into ASB. There's a chance that adding ASB support to the library might be relatively easy. It really depends on how different ASB is from AINB.

**Reply by MediaMoots** [reply]
Thnk you! Sounds good, looking forward to it.
